render.js

```diff
@@ -0,0 +1,50 @@
+// render :: (string, string) -> Node
+function render(domId, content) {
+  const el = document.getElementById(domId).innerHTML = content;
```

Comment:
`const el = document.getElementById(domId).innerHTML = content`, 
OMG , @jeremyfourna , oh.. what is this??? my eyes hurt.